grasa_event_locator/templates/search/search.html

```diff
@@ -247,6 +247,12 @@ <h2 class="mb-0"><button class="btn btn-link collapsed" type="button" data-toggl
         }
     }
     initMap();
+    
+    //replaces the + in an address with a space
+    $(".addressFix").each(function() {
+        var text = $(this).text();
+        $(this).text(text.replace(/\+/g, ' ')); 
```

Comment:
Would it be better to replace with '' instead of ' '?

Reply:
idk